warhorn/warhorn.py

```diff
@@ -1107,6 +1071,26 @@ def get_base_path(node_name="warriorframework", **kwargs):
     return base_path, node
 
 
+def activate_virtualenv(node, destination, logfile, print_log_name):
+    '''Activate virtual environment to add dependencies
+    '''
+    ve_name = get_attribute_value(node, 'name')
+    ve_loc = get_attribute_value(node, 'location')
+    ve_dest = destination + ve_name
+    print_info("ve_name: "+ve_name, logfile, print_log_name)
+    print_info("destination: "+ve_dest, logfile, print_log_name)
+    try:
+        venv_cmd = os.path.expanduser(ve_loc)
+        subprocess.check_call([venv_cmd, "--system-site-packages", ve_dest])
+        venv_file = "{}/bin/activate_this.py".format(ve_dest)
+        execfile(venv_file, dict(__file__=venv_file))
+        return True
+    except Exception as e:
+        print_error("Activating virtual env at {} resulted in exception {}".format(
+                                                ve_dest, e), logfile, print_log_name)
+        return False
+
```

Comment:
Exit value should be set to 1 when activation fails.
use-case: Activation will fail when giving no or invalid value for 'location' attrib

Reply:
Overall we need a result summary or result html for warhorn. This comment is out of scope of this review.

Reply:
@venkat1077 done the changes. Please check and approve.